src/Numerics/ODESolvers/ODESolvers.jl

```diff
@@ -165,5 +165,6 @@ include("StrongStabilityPreservingRungeKuttaMethod.jl")
 include("AdditiveRungeKuttaMethod.jl")
 include("MultirateInfinitesimalStepMethod.jl")
 include("MultirateRungeKuttaMethod.jl")
+include("SplitExplicitMethod.jl")
```

Comment:
This is very ocean specific, should it live here?

Reply:
We spoke about this a few weeks ago and you said we could stick it here, but we can also move it to the Ocean directory

Reply:
Whatever you think makes the most sense. My future me will probably have a third opinion...

Reply:
Okay, I'll leave it with the timesteppers for now, because I want you guys to consider it when updating the rest of the timesteppers (for instance the unified `dt` interface change from a month or so ago)